repos/system_upgrade/common/actors/checkleftoverpackages/actor.py

```diff
@@ -0,0 +1,20 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import checkleftoverpackages
+from leapp.models import InstalledUnsignedRPM, LeftoverPackages, TransactionCompleted
+from leapp.tags import IPUWorkflowTag, RPMUpgradePhaseTag
+
+
+class CheckLeftoverPackages(Actor):
+    """
+    Check if there are any RHEL 7 packages present after upgrade.
+
+    Actor produces message containing these packages. Message is empty if there are no el7 package left.
```

Comment:
@tomasfratrik update please the description. as it has been moved to common repo, the purpose is to detect leftover packages from the old system

Reply:
Missed that description, should be fixed!